### PR TITLE
Update user-guide-content-moved.md

### PR DESCRIPTION
**Pull request by guzj11**
update user-guide-content-move.md to sync with en 
and add the translation.

### CODE REVIEW

File: content/zh/includes/user-guide-content-moved.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<!--
+The topics in the [User Guide](/docs/user-guide/) section of the Kubernetes docs
+are being moved to the [Tasks](/docs/tasks/), [Tutorials](/docs/tutorials/), and
+[Concepts](/docs/concepts) sections. The content in this topic has moved to:
+-->
+kubernetes文档中，[使用手册](/docs/user-guide/)部分中的主题被移动到[任务](/docs/tasks/)，[教程](/docs/tutorials/)，[概念](/docs/concepts)部分。主题内容已移至:
```
Comment (tengqm) [inline content/zh/includes/user-guide-content-moved.md:6]:
```suggestion
Kubernetes文档中[使用手册](/zh/docs/user-guide/)部分中的主题被移动到
[任务](/zh/docs/tasks/)、[教程](/zh/docs/tutorials/)和[概念](/zh/docs/concepts)节。
本主题内容已移至:

```

Comment (guzj11) [reply content/zh/includes/user-guide-content-moved.md]: good suggestion!

**Pull request merged**